packages/images-plugin/lib/plugin.js

```diff
@@ -2,9 +2,9 @@
 
 import _ from 'lodash';
 import {errors} from 'appium/driver';
-import BasePlugin from 'appium/plugin';
+import {BasePlugin} from 'appium/plugin';
```

Comment:
is this a breaking change?

Reply:
no... in what?

Reply:
I mean that BasePlugin is not a default import anymore. Then this line is going to fail in all other clients which were importing the class like that

Reply:
right, but if so, they would fail anyway, because they would still have to change to `appium/plugin` from `@appium/base-plugin`.  

either way: the default export is still `BasePlugin`. I just changed it here because I like named exports.